src/nameserver/system_table.h

```diff
@@ -49,11 +50,11 @@ enum class SystemTableType {
     kGlobalVariable = 3,
     kDeployResponseTime,
     kUser,
+    kObject,
```

Comment:
`object` is too general, pick a better name for privilege

Reply:
What about `ObjectPrivilege`?